docs/root/start/sandboxes/golang_network.rst

```diff
@@ -0,0 +1,70 @@
+.. _install_sandboxes_golang_network:
+
+Golang network filter
+=====================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`netcat <start_sandboxes_setup_netcat>`
+        Used to send ``TCP`` data.
+
+In this example, we show how the `Golang <https://go.dev/>`_ network filter can be used with the Envoy
+proxy.
+
+The example demonstrates a Go plugin that can process tcp data stream directly.
```

Comment:
i still think we should show a bit more in this sandbox, but at least we want an explanation here of what the sandbox does

something like 

> The Go plugin proxies TCP network connections. Proxied responses are prefixed with "hello, ".